librz/include/rz_crypto.h

```diff
@@ -11,6 +11,9 @@ extern "C" {
 
 RZ_LIB_VERSION_HEADER(rz_crypto);
 
+#define RZ_CRYPTO_DIRECTION_ENCRYPT 0
```

Comment:
what are these? Some comments explaining how/when to use them?

Reply:
it's some magic that was hardcoded in the code. i'll change them to ENUM